ros_ethercat_model/launch/joint_state_publisher.launch

```diff
@@ -5,4 +5,7 @@
   <param name="joint_state_controller/publish_rate" value="$(arg publish_rate)"/>
   <!-- spawn controller -->
   <node name="joint_state_controller_spawner" pkg="controller_manager" type="spawner" output="screen" args="joint_state_controller" respawn="true"/>
+
+  <!-- Publish underactuation error -->
+  <node name="error_reporter" pkg="sr_error_reporter" type="error_reporter" output="screen"/>
```

Comment:
I would move this node to a higher level launchfile:
- I don't think that we want this (quite low level) repository to have dependencies on the sr_error_reporter package.
- ros_ethercat repo is used for any of Shadow's etherCAT devices, not only hand E. So we shouldn't assume that there will be underactuated joints.

Reply:
Moved to srhand.launch in https://github.com/shadow-robot/sr_interface/pull/611/commits/82c7fdb3526dae95a1d642631197c11060685dab